packages/node/src/typescript.ts

```diff
@@ -450,17 +450,14 @@ export function fixConfig(
   delete config.compilerOptions.tsBuildInfoFile;
   delete config.compilerOptions.incremental;
 
-  // Target esnext output by default (instead of ES3).
   // This will prevent TS from polyfill/downlevel emit.
   if (config.compilerOptions.target === undefined) {
     // See https://github.com/tsconfig/bases/tree/main/bases
     let target: string;
     if (nodeVersionMajor >= 16) {
       target = 'ES2021';
-    } else if (nodeVersionMajor >= 14) {
-      target = 'ES2020';
```

Comment:
We need to leave this until August 9th

https://vercel.com/changelog/node-js-12-is-being-deprecated